src/pages/search/registration_search/filters/SearchForInstitutionFacetListItem.tsx

```diff
@@ -0,0 +1,82 @@
+import { Autocomplete, ListItem } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSearchForOrganizations } from '../../../../api/hooks/useSearchForOrganizations';
+import {
+  AutocompleteListboxWithExpansion,
+  AutocompleteListboxWithExpansionProps,
+} from '../../../../components/AutocompleteListboxWithExpansion';
+import { AutocompleteTextField } from '../../../../components/AutocompleteTextField';
+import { OrganizationRenderOption } from '../../../../components/OrganizationRenderOption';
+import { dataTestId } from '../../../../utils/dataTestIds';
+import { useDebounce } from '../../../../utils/hooks/useDebounce';
+import { getLanguageString } from '../../../../utils/translation-helpers';
+
+interface SearchForInstitutionFacetRowProps {
+  onSelectInstitution: (institutionId: string) => void;
+}
+
+const defaultOrganizationSearchSize = 10;
+
+export const SearchForInstitutionFacetListItem = ({ onSelectInstitution }: SearchForInstitutionFacetRowProps) => {
+  const { t } = useTranslation();
+
+  const [searchQuery, setSearchQuery] = useState('');
+  const [searchSize, setSearchSize] = useState(defaultOrganizationSearchSize);
+  const debouncedQuery = useDebounce(searchQuery);
+  const institutionSearchQuery = useSearchForOrganizations({ query: debouncedQuery, results: searchSize });
+
+  const options = institutionSearchQuery.data?.hits ?? [];
+
+  return (
+    <ListItem sx={{ p: '0.25rem 0.5rem' }}>
+      <Autocomplete
```

Comment:
Ville helst åpnet options her bare når brukeren har gjort et søk, men får ikke til å droppe automatisk åpning for en eller annen grunn... Heller ikke med `openOnFocus={false}` 🤔